reference/concepts/default_arguments.md

```diff
@@ -1,3 +1,5 @@
-A default argument is an argument to a function that a programmer is not required to specify. In most programming languages, functions may take one or more arguments. Usually, each argument must be specified in full (this is the case in the C programming language). Later languages (for example, in C++) allow the programmer to specify default arguments that always have a value, even if one is not specified when calling the function.
+# Default arguments
+
+A default argument is an argument to a [function](./functions.md) that a programmer is not required to specify. In most programming languages, functions may take one or more arguments. Usually, each argument must be specified in full (this is the case in the C programming language). Later languages (for example, in C++) allow the programmer to specify default arguments that always have a value, even if one is not specified when calling the function.
 
 Reference: [https://en.wikipedia.org/wiki/Default_argument](https://en.wikipedia.org/wiki/Default_argument)
```

Comment:
Was wiki the source? If you, per #163, can you pull the link through https://zbib.org/ and use the result instead? It will probably generate something like this, but then make the link clickable :)

```markdown
## References <!-- (or sources) -->

‘Default Argument’. Wikipedia, 20 Nov. 2019. Wikipedia, [https://en.wikipedia.org/w/index.php?title=Default_argument&oldid=927101957](https://en.wikipedia.org/w/index.php?title=Default_argument&oldid=927101957).
```